### DISCUSSION TITLE
"Pending or running Renovate job already exists for this repo" when nothing is running

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#26580] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I want to trigger a Renovate run to generate a PR for a particular update. When I check the relevant box and click "Create/Rebase", I get an error:

> Pending or running Renovate job already exists for this repo

This is despite there being no jobs running, as reported by the "Recent jobs" list. There is a "pending" hourly job at the top, but the last run was several minutes ago. I've refreshed several times and seen no difference. I've also tried triggering my desired update several times and I get the same error, as above, each time.

I'm not sure how to reproduce this in an example repo...

### Relevant debug logs

I can't see anything relevant in the logs for the most recent job.

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by rarkins** [comment]
Please take a screenshot or loom so we can look into it. You also need to identify the repo somehow

**Reply by allanlewis** [reply]
I'm not sure what I could demonstrate with a screenshot beyond what I've written. The repo is private, unfortunately.

In any case, this is no longer happening, I'm not sure what changed but it was 100% reproducible when I wrote this.
